src/Handlers/vHandler.cpp

```diff
@@ -204,16 +204,17 @@ vHandler * getHandler(pugi::xml_node node) {
 		ret = new acSyntheticTurbulence;
     } else if (name=="Run") {
 		output("Skipping 'Run' element");
-    }else {
+        ret = new Callback();
```

Comment:
because: https://github.com/CFD-GO/TCLB/blob/v6.1_candidate/src/Handlers.h#L17

It does fault with NULL pointer. I think that returning NULL where pointer-to-object is expected is design error. 

I could change it to 
```C++

		if (hand != NULL) {
     		        hand->solver = solver_;
			int ret = hand->Init();
			if (ret) {
				delete hand;
				hand = NULL;
			}
		}
		ref = new int;
		*ref=1;
debug0("Handler shared pointer: create\n");
```

but other methods of Handler might expect it to be valid object somewhere